src/exercise/02/02.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+
+def start():
+    print(
+        """
+    Welcome to Pokémon Finder!
+    --------------------------
+  """
+    )
+
+
+def get_choice():
+    return input("Which Pokémon do you want to know about...\n")
+
+
+def fetch_pokemon(choice):
+    req = requests.get("https://pokeapi.co/api/v2/pokemon/" + choice.lower())
+
+    if req.status_code == requests.codes.ok:
+        print(req.text)
+    else:
+        req.raise_for_status()
```

Comment:
I wish this was request default behavior, in python, using exceptions to do flow control in python is actually considered good in most of cases.

Reply:
Yeah, I definitely wish this acted a little more like a JS Promise to help control the flow and readability, but I respect that Python has it's own strategy.